tao_compiler/mlir/disc/transforms/disc_transform_legalize_to_loop.cc

```diff
@@ -265,6 +287,23 @@ LogicalResult DiscTransformLegalizeToLoopPass::handleCpuFusionOp(
     // skip non-transform-based fusion pattern.
     return success();
   }
+  auto& bypassMap = bypassCodegenPatternNameMap();
+  auto it = bypassMap.find(getFusionName(fusionOp).str());
```

Comment:
nit: is fusion name stable?

Reply:
Yes, the name is generated using a fixed schema.